### PR TITLE
switch to crypto library

### PR DESCRIPTION
**Pull request by emelalkim**
md5 library was using too much memory for large files

### CODE REVIEW

**Review by pieper — COMMENTED** [review]
Hi Emel - question: looking on npm it says that crypto is deprecated.  Are you sure we shouldn't be using crypto-js or something else?

File: plugins/CouchDB.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const dcmjs = require('dcmjs');
 const Axios = require('axios');
 const http = require('http');
 const fs = require('fs');
-const crypto = require('crypto');
+const CryptoJS = require('crypto');
```
Comment (pieper) [inline plugins/CouchDB.js:11]: Shouldn't this be `require('cryptojs')`?

**Review by pieper — APPROVED** [review]
👍

**Comment by pieper** [comment]
Let me know when this is ready to merge.

**Comment by emelalkim** [comment]
Unfortunately, this one is crashing the memory too. 
I'm closing this PR so that you don't get notifications every time I push changes, I cannot test it on my local so I need to push every change. 
I'll try to find another library and open the PR again.

**Pull request closed without merging**